modules/ROOT/pages/java/java-instanceof.adoc

```diff
@@ -6,14 +6,30 @@ endif::[]
 
 
 
+The Java Module provides a `validate-type` operation that tests whether a given object is an
+instance of a given class. It can also be configured to accept or reject SubTypes of the
+given class.
 
+== Validate Type Operation
+
+The following examples checks whether the variable `vars.jhon` belongs to the class
```

Comment:
Rephrase it, doesn't make sense to talk about a variable without giving a context of what there is in the variables.

Reply:
added java code and context